app/config/wp-demo/install.sh

```diff
@@ -53,7 +53,9 @@ wp eval '$home = get_page_by_title("Welcome to CiviCRM with WordPress"); update_
 
 wp plugin activate civicrm
 wp plugin activate civicrm-demo-wp
-wp plugin install civicrm-admin-utilities && wp plugin activate civicrm-admin-utilities
+wp plugin install civicrm-admin-utilities --activate
+wp plugin install gutenberg
```

Comment:
Will this line break anything when 5.0 is released and Gutenberg isn't an extension anymore?

Reply:
They say line 57 should not break once it's no longer a plugin.  We at most should get a 'plugin not found' message.  I added simply for ease of use.  I imagine that we may have to remove when 5.0 is released.

We could remove now and have people manually install.   I am flexible.  I did think having it installed but not active would be better and encourage testing with it.